src/services/worker/tasks/liquidation.task.ts

```diff
@@ -38,31 +38,29 @@ export class LiquidationTask {
     }
     LiquidationTask.isProcessLocked = true;
 
-    const latestSyncedBlockNumber = await this._systemService.get(
-      SystemType.GENERAL_LAST_BLOCK_NUMBER,
-    );
+    try {
+      const latestSyncedBlockNumber = await this._systemService.get(
+        SystemType.GENERAL_LAST_BLOCK_NUMBER,
+      );
 
-    const latestBlockNumber =
-      await this._web3Provider.web3.eth.getBlockNumber();
+      const latestBlockNumber =
+        +(await this._web3Provider.web3.eth.getBlockNumber());
```

Comment:
is this casting really necessary? according to web3 docs this call returns Number